include/rocksdb/c.h

```diff
@@ -1114,9 +1114,15 @@ rocksdb_sstfilewriter_create_with_comparator(
     const rocksdb_comparator_t* comparator);
 extern ROCKSDB_LIBRARY_API void rocksdb_sstfilewriter_open(
     rocksdb_sstfilewriter_t* writer, const char* name, char** errptr);
-extern ROCKSDB_LIBRARY_API void rocksdb_sstfilewriter_add(
```

Comment:
Removing an external visible api will break users when they try to upgrade. So, is there a way we can 'deprecate' this in the C api as well, instead of totally removing it? 
I see that we use ROCKSDB_DEPRECATED_FUNC to identify a function as deprecated in C++, but how do we handle it in C and Java API ? (I don't see anything in our C and Java code, hence asking).